### PR TITLE
New getting started

### PR DESCRIPTION
**Pull request by nnenasheva**

### CODE REVIEW

File: user_guide/access_permissions.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+- Access Settings defined for the Role assigned to the User
+
+
+.. _user-guide-user-management-permissions-ownership-type:
+
+Ownership Types
+---------------
+
+Each entity in OroCRM has an :ref:`ownership type <user-guide-entity-management-create-other>` type, which defines the
```
Comment (DimaSoroka) [inline user_guide/access_permissions.rst]: _type_ mantioned two times

File: user_guide/access_permissions.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+.. _user-guide-user-management-permissions-ownership-type:
+
+Ownership Types
+---------------
+
+Each entity in OroCRM has an :ref:`ownership type <user-guide-entity-management-create-other>` type, which defines the 
+level at which permissions will be set for records of the entity.
+
+If the ownership type is set to *"None"*, no authorization is required to see and process the entity and all the users
```
Comment (DimaSoroka) [inline user_guide/access_permissions.rst]: This is not correct to say "no authorization is required"

File: user_guide/access_permissions.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
+.. _user-guide-user-management-permissions:
```
Comment (DimaSoroka) [inline user_guide/access_permissions.rst]: Please provide reference to https://github.com/orocrm/documentation/blob/master/book/security.rst and make sure that content plays nicely together.

File: user_guide/first_steps.rst [hunk 4]
```diff
@@ -0,0 +1,379 @@
+.. _user-guide-started:
+
+Ready, Set, Go
+==============
+
+OroCRM is a highly flexible system that can be tailored to meet the needs of your specific business.
+Major set-up is performed at the back-end during the system integration and described in more details in the
```
Comment (DimaSoroka) [inline user_guide/first_steps.rst]: What is the "back-end" here?

File: user_guide/access_permissions.rst (hunk 3 above)
Comment (nnenasheva) [inline user_guide/access_permissions.rst]: Done. The intro now says

OroCRM provides a very flexible structure of access and permissions management that can be defined from the 
:ref:`security bundle <ubook-security-bundle>` and/or from the UI. 

From the UI you can define ability of a specific user or group of users to view, create, edit or delete 
records of a specific :term:`entity <Entity>` and change its :term:`owner <Owner>`.

File: user_guide/access_permissions.rst (hunk 1 above)
Comment (nnenasheva) [inline user_guide/access_permissions.rst]: fixed

File: user_guide/access_permissions.rst (hunk 2 above)
Comment (nnenasheva) [inline user_guide/access_permissions.rst]: Removed the part. It now says: "If the ownership type is set to *"None"*, all the users
within the OroCRM instance will be able to ..."

File: user_guide/first_steps.rst (hunk 4 above)
Comment (nnenasheva) [inline user_guide/first_steps.rst]: a process or part of the program, not directly accessed by the user

shall I change the work to something else?

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for about an hour for the submission to be processed.

**Comment by nnenasheva** [comment]
duplicated by a newer and better version of the PR

**Pull request closed without merging**